main.nf

```diff
@@ -418,6 +418,7 @@ process SPLIT_VCF_BY_CHROMOSOME {
 }
 
 process VEP_ANNOTATE {
```

Comment:
ANNOTATE_BY_VEP

Reply:
Well technically the input is TSV

Reply:
I would like to use the name of the content, not the format, because half of process are dealing VCF data, and the rest half are dealing TSV data. 

btw if you mean our VEP process takes tsv text files, then maybe this variable name is also confusing, and needs to be renamed..:
![image](https://github.com/user-attachments/assets/66c60f82-f9e4-4d66-8575-cf3c9a5becad)